core/src/main/scala/pureconfig/error/ConfigReaderFailure.scala

```diff
@@ -69,8 +84,12 @@ final case object CannotConvertNull extends ConfigReaderFailure {
  * @param because the reason why the conversion was not possible
  * @param location an optional location of the ConfigValue that raised the
  *                 failure
+ * @param path an optional path to the value that couldn't be converted
```

Comment:
I wish there were a good way to disambiguate "path" from the typical filesystem sense in which it's used. I don't have any good ideas. Do you?
(Ditto on 56.)